aws/provider.go

```diff
@@ -393,295 +394,295 @@ func Provider() terraform.ResourceProvider {
 			"aws_dx_hosted_private_virtual_interface_accepter": resourceAwsDxHostedPrivateVirtualInterfaceAccepter(),
 			"aws_dx_hosted_public_virtual_interface":           resourceAwsDxHostedPublicVirtualInterface(),
 			"aws_dx_hosted_public_virtual_interface_accepter":  resourceAwsDxHostedPublicVirtualInterfaceAccepter(),
-			"aws_dx_lag":                                       resourceAwsDxLag(),
```

Comment:
The TravisCI failure below is likely because you are using Go 1.10 or earlier locally, which will `go fmt` files slightly differently. If you upgrade to Go 1.11, this should revert back.

```
$ make test
==> Checking that code complies with gofmt requirements...
gofmt needs running on the following files:
./aws/provider.go
You can use the command: `make fmt` to reformat code.
make: *** [fmtcheck] Error 1
The command "make test" exited with 2.
```

Reply:
That is indeed the case and I'll be working from 1.11+ going forward.